tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -1042,19 +1087,38 @@ ROOT::Experimental::Internal::RMiniFileReader::GetNTupleProper(std::string_view
 
    ReadBuffer(&key, sizeof(key), key.GetSeekKey());
    offset = key.GetSeekKey() + key.fKeyLen;
-   RTFNTuple ntuple;
-   if (key.fObjLen > sizeof(ntuple)) {
-      return R__FAIL("invalid anchor size: " + std::to_string(key.fObjLen) + " > " + std::to_string(sizeof(ntuple)));
+
+   if (key.fObjLen < sizeof(RTFNTuple)) {
+      return R__FAIL("invalid anchor size: " + std::to_string(key.fObjLen) + " > " + std::to_string(sizeof(RTFNTuple)));
```

Comment:
```suggestion
   if (key.fObjLen < sizeof(RTFNTuple)) {
      return R__FAIL("invalid anchor size: " + std::to_string(key.fObjLen) + " < " + std::to_string(sizeof(RTFNTuple)));
```
The below comment explains why the object length can be larger than the size now, but why can it not be smaller now when this was the case before?

Reply:
In principle, a future anchor version could be smaller. We explicitly forbid it here, based on the convention that we only ever should add information to the anchor.  Not sure, does it answer the question?